index.html

```diff
@@ -566,7 +566,12 @@
                 <ul>
                     <li>The user agent has determined that a pointer is unlikely to continue to produce events (for example, because of a hardware event).</li>
                     <li>After having fired the <code>pointerdown</code> event, if the pointer is subsequently used to manipulate the page viewport (e.g. panning or zooming).</li>
+                    <li>Immediately before <a href="https://html.spec.whatwg.org/multipage/dnd.html#initiate-the-drag-and-drop-operation">drag operation starts</a> [[!HTML]],
+                     for the pointer that caused the drag operation.</li>
                 </ul>
+<div class="note">If the start of the drag operation is prevented through any means
```

Comment:
Nits:
- Can we move the Note to inside the last list element (`<li>...</li>`)?
- Raw text indentation.